docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -1897,3 +1897,300 @@ pip install gremlinpython
 pip install aiohttp
 python example.py
 ----
+
+[[gremlin-go]]
```

Comment:
looks like this was added to the end of the document. the languages are in alphabetical order now.